config.js

```diff
@@ -0,0 +1,19 @@
+
+module.exports = {
+    extends: ["group:allNonMajor", "config:recommended"],
+    timezone: "Europe/Berlin",
+    dependencyDashboard: true,
+    logFile: "renovate.log",
+    repositories: ['mirek163/api-layer'],
+    baseBranches: ["v2.x.x-renovate","v3.x.x-renovate"],
+    assignees: ["mirek163"],
+    reviewers: ["mirek163"],
```

Comment:
Please, remove yourself from assignees and reviewers. And let's keep it empty/commented out for now.